aea/cli/check_packages.py

```diff
@@ -307,7 +307,7 @@ def check_dependencies(
     data = unified_yaml_load(configuration_file)
 
     def _add_package_type(package_type: PackageType, public_id_str: str) -> PackageId:
-        return PackageId(package_type, PublicId.from_str(public_id_str))
+        return PackageId(package_type, PublicId.from_str(public_id_str).without_hash())
```

Comment:
One might argue we should check hashes as well, but as I stated these are not present in the .yaml of the component itself so cannot be retrieved from the configuration file itself here: 
https://github.com/valory-xyz/open-aea/blob/3b27e783960ffe259b4cb490ad96f74821444b8a/aea/cli/check_packages.py#L298-L300

Reply:
reverted this change